compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -224,6 +224,7 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
       // compute op into the dispatch region, so that we can run additional
       // transformations afterwards with a simple region and without bothering
       // producers.
+      .addPass(IREE::Flow::createTopLevelSCFToCFGPass)
```

Comment:
Why do we move it between `FormScalarDispatchPass` and `FormDispatchRegionsPass`? I thought that it should happen maybe right before `createFusionOfTensorOpsPass`?

Reply:
`FormDispatchRegionsPass` does not support `scf` operations yet. I was going to continue pushing the `scf` support forward pass-by-pass.